target/linux/generic/hack-6.6/781-usb-net-rndis-support-asr.patch

```diff
@@ -30,7 +30,7 @@ Signed-off-by: Daniel Golle <daniel@makrotopia.org>
  /*-------------------------------------------------------------------------*/
  
  static const struct usb_device_id	products [] = {
-@@ -666,6 +676,36 @@ static const struct usb_device_id	produc
+@@ -676,6 +686,40 @@ static const struct usb_device_id	produc
```

Comment:
`+@@ -676,6 +686,36 @@ static const struct usb_device_id	produc`

Reply:
I believe the most recent commit fixes this. I do not like directly editing patches :smile: